model/labels/labels.go

```diff
@@ -248,6 +248,16 @@ func (ls Labels) WithoutEmpty() Labels {
 	return ls
 }
 
+// ByteSize returns the approximate size of the labels in bytes.
+func (ls Labels) ByteSize() int {
+	var size = 0
+	for _, l := range ls {
+		size += len(l.Name) + len(l.Value) + 2*int(unsafe.Sizeof(""))
+	}
+	size += int(unsafe.Sizeof([]Labels{}))
```

Comment:
This is likely good enough:

```suggestion
		size += len(l.Name) + len(l.Value)
	}
```